engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/content/IReportContent.java

```diff
@@ -61,17 +61,25 @@ public interface IReportContent {
 	/**
 	 * Gets the toc tree of this report content.
 	 *
+	 * @param format
+	 * @param locale
+	 *
 	 * @return the TOC Tree
 	 */
 	ITOCTree getTOCTree(String format, ULocale locale);
 
 	/**
-	 * return root content
+	 * Get the root content
 	 *
-	 * @return
+	 * @return Return the root content
```

Comment:
For the future it is not needed to add Return to @return

Reply:
I will think on it!